modules/src/wiki.py

```diff
@@ -4,9 +4,10 @@ def process(input, entities):
     output = {}
     try:
         query = entities['wikipedia_search_query'][0]['value']
+        page = wikipedia.page(query)
         data = wikipedia.summary(query, sentences=1)
```

Comment:
I've made some refactoring so if you could fork and send a PR again, that'd be great!
Making fewer calls is our top priority. Take the full summary and wrap it like [this](https://github.com/swapagarwal/JARVIS-on-Messenger/blob/master/modules/src/movie.py), setting url in post_text.